### ISSUE TITLE
Support for azurerm_cosmosdb_gremlin_database autoscale throughput

### ISSUE THREAD

**Issue by CasperNaudts** [terraform-providers/terraform-provider-azurerm#6517]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Description

In azure cosmosdb gremlin there is an option to auto scale the throughput when creating the resource. In terraform this option is as far as I know not available

### New or Affected Resource(s)

* azurerm_cosmosdb_gremlin_database

### Potential Terraform Configuration

```hcl
resource "azurerm_cosmosdb_gremlin_database" "production" {
    name                = "production"
    resource_group_name = azurerm_resource_group.data.name
    account_name        = azurerm_cosmosdb_account.account.name
    min_throughput      = 400
    max_throughput      = 1000
}
```

### References

* #0000

**Comment by StefanSchoof** [comment]
Sound similar to #6164

**Comment by katbyte** [comment]
closing as this seems to have been fixed by #7773
